pkg/scheduler/apis/config/v1beta1/defaults.go

```diff
@@ -140,6 +140,11 @@ func SetDefaults_KubeSchedulerConfiguration(obj *v1beta1.KubeSchedulerConfigurat
 	// Use the default LeaderElectionConfiguration options
 	componentbaseconfigv1alpha1.RecommendedDefaultLeaderElectionConfiguration(&obj.LeaderElection)
 
+	if obj.UnschedulableQTimeInterval == nil {
+		val := int64(60)
+		obj.UnschedulableQTimeInterval = &val
```

Comment:
You can use `pointer.Int64Ptr(60)`. (please help to correct the logic of PodInitial*Seconds below)

Reply:
Yes and thanks! Will update this and PodInitial*Seconds below.

Reply:
should we also reference `pkg/scheduler/internal/queue.DefaultUnschedulableQTimeInterval` instead of hardcoding 60 here? that pattern isn't followed for other fields though

Reply:
> should we also reference `pkg/scheduler/internal/queue.DefaultUnschedulableQTimeInterval` instead of hardcoding 60 here?

I guess no, in the perspective of pkg dependency.